internal/backends/mcu/Cargo.toml

```diff
@@ -48,6 +48,8 @@ pin-weak = { version = "1", default-features = false }
 rgb = "0.8.27"
 scoped-tls-hkt = { version = "0.1", optional = true }
 winit = { version = "0.26.0", default-features = false, optional = true, features = ["x11"] }
+copypasta = { version = "0.8.1", optional = true, default-features = false }
```

Comment:
I think we should get rid of the simulator sooner than later so this kind of dependency don't need to be added.

But for now let's just do it like that.